src/test/regress/sql/multi_schema_support.sql

```diff
@@ -983,6 +983,34 @@ BEGIN;
     ALTER SCHEMA bar RENAME TO foo;
 ROLLBACK;
 
+-- verify that Citus uses current user's metadata connection to propagate table deps since sc1, which is one of the deps of table s1, is created in the same transaction.
+BEGIN;
+    CREATE SCHEMA sc1;
+    CREATE SEQUENCE sc1.seq;
+    CREATE TABLE sc1.s1(id int default(nextval('sc1.seq')));
+    SELECT create_distributed_table('sc1.s1','id');
+COMMIT;
+DROP SCHEMA sc1 CASCADE;
+
+-- verify that Citus uses current user's metadata connection to propagate table deps since seq1, which is one of the deps of table s1, is created in the same transaction.
```

Comment:
we do not propagate sequences, so cannot track them. This will still use outside connection. Should we care?

Reply:
I think not. The visibility problem only occurs when we propagate an object via metadata connection, and then later a dependency depends on that object. (It is invisible to outside connection). We alreday resolve that by tracking all propagated objects in the current transaction. (we use metadata connection even if it fails due to acl error)